Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.xaml

```diff
@@ -1,14 +1,14 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls"
-                    xmlns:controls ="using:Microsoft.Toolkit.Uwp.UI.Controls">
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
```

Comment:
Any reason why both local and controls point to the same namespace?

Reply:
That's an old code but Xaml Styler helped to show the issue in this PR :)

Reply:
From what I can see, this is in the new code as well, just the order has changed.

Reply:
😄  yeah, I know it exist in the new code that's why I mentioned issue, I thought you were asking why you added a new namespace 😃  will fix it with the new changes.